packages/rocketchat-federation/server/index.js

```diff
@@ -0,0 +1,17 @@
+// Federation export
+const Federation = {
+	enabled: false,
+	privateKey: null,
+	publicKey: null,
+	usingHub: null,
+	uniqueId: null,
+	localIdentifier: null,
+	peerDNS: null,
+	peerHTTP: null,
+	peerClient: null,
+	peerServer: null,
+};
+
+export {
```

Comment:
if exporting as suggested above this can be removed

Reply:
👍